### PR TITLE
pets api JSON testing

### PR DESCRIPTION
**Pull request by jeffsmw**
@thaisamarinho can you check this over 😄

### CODE REVIEW

File: spec/controllers/api/v1/pets_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::PetsController, type: :controller do
+  render_views
+
+  describe 'when map (:top param) is present' do
+    it 'responds with 200 code' do
+      get :index, { :format => :json, :top => '49' }
```
Comment (thaisamarinho) [inline spec/controllers/api/v1/pets_controller_spec.rb]: Jeff, i would change this -> { :format => :json, :top => '49' } to -> {format: :json, top: '49'}

File: spec/controllers/api/v1/pets_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      get :index, format: :json
+      expect(response).to be_success
+    end
+
+    it 'returns a JSON list of correct length' do
+      FactoryGirl.create_list(:pet, 5)
+      get :index, format: :json
+      # expect(response).to be_success
+      # body = JSON.parse(response.body)
+      # byebug
```
Comment (thaisamarinho) [inline spec/controllers/api/v1/pets_controller_spec.rb]: I would remove all the comments

File: spec/controllers/api/v1/pets_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      get :index, { :format => :json, :top => '49' }
+      expect(response).to be_success
+    end
+
+    it 'returns a valid pet' do
+      FactoryGirl.create :pet, name: 'Valid'
+      get :index, { :format => :json, :top => 49.5,
+                                      :bottom => 49.0,
+                                      :left => -124.0,
+                                      :right => -122.0 }
```
Comment (thaisamarinho) [inline spec/controllers/api/v1/pets_controller_spec.rb]: Same comment as above, i would change all the => for the ::

File: spec/controllers/api/v1/pets_controller_spec.rb (hunk 1 above)
Comment (thaisamarinho) [reply spec/controllers/api/v1/pets_controller_spec.rb]: and the "49" just 49 to keep the pattern.

**Review by thaisamarinho — COMMENTED** [review]
Hey Jeff, good job! i made some small comments on your code. Thanks!

**Comment by jeffsmw** [comment]
@thaisamarinho Code has been updated to be consistent and comments removed!

**Comment by jeffsmw** [comment]
@whilestevego Ready for a review!

**Review by whilestevego — APPROVED** [review]
Merge it if tests pass!

**Comment by whilestevego** [comment]
Merge it for you, @jeffsmw! Using the squash and merge strategy.

**Pull request merged**